tsdb/engine.go

```diff
@@ -60,6 +74,21 @@ func NewEngine(path string, walPath string, options EngineOptions) (Engine, erro
 	// Only bolt-based backends are currently supported so open it and check the format.
 	var format string
 	if err := func() error {
+		// if it's a dir then it's a tsm1 engine
```

Comment:
Ok, I'll update it to check if there are tsm files and check the magic number of the first one. Will do that later this week, but I've made a note of it.